yt_dlp/extractor/mediaset.py

```diff
@@ -217,23 +125,25 @@ def _check_drm_formats(self, tp_formats, video_id):
 
     def _real_extract(self, url):
         guid = self._match_id(url)
-        tp_path = 'PR1GhC/media/guid/2702976343/' + guid
+        tp_path = f'PR1GhC/media/guid/2702976343/{guid}'
         info = self._extract_theplatform_metadata(tp_path, guid)
 
         formats = []
         subtitles = {}
         first_e = geo_e = None
         asset_type = 'geoNo:HD,browser,geoIT|geoNo:HD,geoIT|geoNo:SD,browser,geoIT|geoNo:SD,geoIT|geoNo|HD|SD'
         # TODO: fixup ISM+none manifest URLs
-        for f in ('MPEG4', 'M3U'):
+        for f in ('MPEG4', 'MPEG-DASH', 'M3U'):
             try:
                 tp_formats, tp_subtitles = self._extract_theplatform_smil(
-                    update_url_query('http://link.theplatform.%s/s/%s' % (self._TP_TLD, tp_path), {
+                    update_url_query(f'http://link.theplatform.{self._TP_TLD}/s/{tp_path}', {
                         'mbr': 'true',
                         'formats': f,
                         'assetTypes': asset_type,
-                    }), guid, 'Downloading %s SMIL data' % (f.split('+')[0]))
+                    }), guid, f'Downloading {f.split("+")[0]} SMIL data')
             except ExtractorError as e:
+                if e.orig_msg == 'None of the available releases match the specified AssetType, ProtectionScheme, and/or Format preferences':
+                    e.orig_msg = 'Content behind paywall and DRM'
```

Comment:
```suggestion
                    e.orig_msg = 'This video is DRM protected'
```
is the standard message used